packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -46,6 +46,30 @@ export default {
       values: "object/values",
     },
 
+    Math: {
+      acosh: "math/acosh",
+      asinh: "math/asinh",
+      atanh: "math/atanh",
+      cbrt: "math/cbrt",
+      clz32: "math/clz32",
+      cosh: "math/cosh",
+      expm1: "math/expm1",
+      fround: "math/fround",
+      hypot: "math/hypot",
+      imul: "math/imul",
+      log10: "math/log10",
+      log1p: "math/log1p",
+      log2: "math/log2",
+      sign: "math/sign",
+      sinh: "math/sinh",
+      tanh: "math/tanh",
+      trunc: "math/trunc",
+      iaddh: "math/iaddh",
+      isubh: "math/isubh",
+      imulh: "math/imulh",
+      umulh: "math/umulh",
```

Comment:
Some of these (e.g. `umulh`) are not part of the standard (https://tc39.github.io/ecma262/)